application/src/test/java/dev/aws101/TodoApplicationTests.java

```diff
@@ -1,14 +0,0 @@
-package dev.aws101;
-
-import org.junit.jupiter.api.Test;
-
-import org.springframework.boot.test.context.SpringBootTest;
```

Comment:
This test kept failing because somewhere during the Spring container startup it was trying to contact Cognito and failing. We need to find a way to disable that during tests. I took the shortcut by deleting this test...